syntax = "proto3";

enum Status {
    FAILED = 0;
    SUCCESS = 1;
  }

service RegistryServerService {
    rpc RegisterServer (RegisterServerRequest) returns (RegisterServerResponse) {}
  }

message RegisterServerRequest {
    string server_name = 1;
  }

message RegisterServerResponse {
    Status status = 1;
  }